src/lib/jobRunner.ts

```diff
@@ -11,15 +11,15 @@
  * @returns {Function} returns a function that accepts the parameters
  * to be passed to the "userFunc" function
  */
-const jobRunner = userFunc => e => {
+const jobRunner = (userFunc: Function) => (e: { data: any[] }) => {
```

Comment:
What do you guys think about using `e: MessageEvent` here instead? Per [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Worker) cc @alewin @IljaDaderko

Reply:
Makes sense to me 👍

Reply:
Oh cool, seems that OP just missed it here :)

Reply:
yes, it seems right! 👌

Reply:
@grdnrt @alewin @IljaDaderko updated, could someone take a look ?)